packages/alfa-rules/src/sia-r111/rule.ts

```diff
@@ -54,11 +61,29 @@ export default Rule.Atomic.of<Page, Element>({
  * @public
  */
 export namespace Outcomes {
-  export const HasSufficientSize = Ok.of(
-    Diagnostic.of("Target has sufficient size"),
-  );
+  export const HasSufficientSize = (name: string, box: Rectangle) =>
+    Ok.of(WithBoundingBox.of("Target has sufficient size", name, box));
+
+  export const HasInsufficientSize = (name: string, box: Rectangle) =>
+    Err.of(WithBoundingBox.of("Target has insufficient size", name, box));
+
+  export const IsUserAgentControlled = (name: string) =>
+    Ok.of(WithName.of("Target is user agent controlled", name));
```

Comment:
This might get a bit annoying downstream if we provide incompatible diagnostics without a differenciating property.

I think for know it might be easier to always send a `WithBouindingBox`. We need to work out a better system for incompatible diagnostics, but that requires more sync and contracts with downstream usage.

Reply:
Done